### PR TITLE
Add default failbacks

### PR DESCRIPTION
**Pull request by paltman**

### CODE REVIEW

**Comment by paltman** [comment]
[Add fallback messages when no data is returned for a given annotation](https://trello.com/c/fBcx80xA/170-add-fallback-messages-when-no-data-is-returned-for-a-given-annotation)

File: src/widgets/AudioWidget.vue [hunk 1]
```diff
@@ -8,7 +8,8 @@
     >
       No audio support
     </audio>
-    <small class="audio-attribution">
+    <div v-if="nowPlaying === null">No audio available</div>
```
Comment (jacobwegner) [inline src/widgets/AudioWidget.vue:11]: Thanks!  I meant to do that but forgot to

File: src/widgets/ScholiaWidget.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="scholia">
-    <div v-if="!lines">No commentary found.</div>
+    <div v-if="lines.length === 0">No commentary found.</div>
```
Comment (jacobwegner) [inline src/widgets/ScholiaWidget.vue:3]: @paltman getting a console error on a passage without scholia:

image


https://deploy-preview-37--explorehomer-dev.netlify.app/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-eng3%3A1.1

**Comment by jtauber** [comment]
The first thing that's clear is that we're completely inconsistent in the styling of the widget fallback.

* different font size
* different indentation
* "No X available" vs "No Y" vs "No Z found"
* period vs no period

My slight preference is the Named Entities variant. I certainly prefer the smaller font size, less indentation, and no period.

Not sure what which I prefer between "available" or "found" or neither.

**Comment by paltman** [comment]
I'll make a component called `<Attribution>` that can have it's own single style.

**Comment by paltman** [comment]
@jtauber everything is now the same font/size/padding/message.  It's a default message in a slot so we can override with specific language.  Just let me know what language you want in what widget/place.

**Comment by jtauber** [comment]
I'd drop the `text-align: center;` and change the padding to `padding: 0 0 0.4rem;`

**Comment by paltman** [comment]
@jtauber fixed

**Pull request merged**